e2e/tests/relay.feature

```diff
@@ -1,5 +1,6 @@
 Feature: Relay Namespace
 
+    # TODO_TECHDEBT(@Olshansk, #180): This test requires you to run `make supplier1_stake && make app1_stake` first
```

Comment:
For the purposes of the E2E test, we also have the option to shortcut #180 by modifying existing step definitions (e.g. `#TheApplicationIsStakedForService`), or adding new step(s) (i.e. "given ..."),  which do this staking against local/dev-net

Reply:
Updating the comment but want to keep the actual change outside the scope of this PR.